packages/react-i18n/src/test/i18n.test.ts

```diff
@@ -346,6 +346,42 @@ describe('I18n', () => {
     });
   });
 
+  describe('#unformatCurrency()', () => {
```

Comment:
TODO: Assuming reviewers are okay with the approach in the utilities, add tests for more unique locales.